Misc/Positions_Area Positions 3 UK.txt

```diff
@@ -50,11 +50,12 @@ London Control TC TIMBA:London Control:120.170:TCST:L:LTC:CTR:-:-:0301:0377:N051
 London Control:London Control:135.800:TCC:L:LTC:CTR:-:-:: ;TC Bandbox
 London Control:London Control:127.950:TCV:L:LTC:CTR:-:-::
 ;Prestwick W2 LAG (MAN)
-North:Scottish Control:128.670:PCN:M:MAN:CTR:-:-:0501:0577:N053.49.11.974:W002.18.59.167
+Penil:Scottish Control:126.870:PCP:M:MAN:CTR:-:-:0501:0577:N053.28.48.603:W003.58.46.843
+NorthEast:Scottish Control:135.700:PCNE:M:MAN:CTR:-:-:0501:0577:N053.49.11.974:W002.18.59.167
 Isle of Man:Scottish Control:133.050:IOM:M:MAN:CTR:-:-:0501:0577:N053.28.48.603:W003.58.46.843
 Sector 29:Scottish Control:133.200:S29:M:MAN:CTR:-:-:0501:0577:N053.31.33.436:W002.32.36.984
```

Comment:
Would this being the same freq as the new PC BBX not conflict when logging on?

Reply:
Indeed, have changed to 118.775